scaleway-core/scaleway_core/api.py

```diff
@@ -168,6 +166,12 @@ def _request(
             verify=not self.client.api_allow_insecure,
         )
 
+        if response.headers.get("x-total-count"):
```

Comment:
Can we have tests for it ? We can use `listServerTypes` as it does not require a token.